frontend/document/credocument.lua

```diff
@@ -99,6 +99,11 @@ function CreDocument:cacheInit()
         G_reader_settings:readSetting("cre_storage_size_factor") or default_cre_storage_size_factor)
 end
 
+function CreDocument.disableCache()
+    cre = cre or CreDocument:engineInit()
+    cre.initCache("", 0, true, 40)
+end
```

Comment:
You plan to use this for other things ?
Depending on the need, other use cases may want to use other parameters, so I'd somehow like to have that in the call site (like we do in the other places we tweak it).
Anyway, if keeping it, add a comment:
`-- To be used only when in a subprocess to not interfere with opened books caches`